### PR TITLE
Update geth

### PR DESCRIPTION
**Pull request by MartinOndejka**

### CODE REVIEW

File: src/app/zeko/default.nix [hunk 1]
```diff
@@ -4,8 +4,8 @@ let
     src = pkgs.fetchFromGitHub {
       owner = "dcspark";
       repo = "go-ethereum";
-      rev = "352d900fea2cd96b5dc1ee08527e39db994f98a5";
-      sha256 = "sha256-emNQSq4Uhi8GM5vTkP8vqPsQbkjnnla/IYFQrFxQbGM=";
+      rev = "daee9a8b65f5ba06c36d38bc5a6ed3609a505c9b";
+      sha256 = pkgs.lib.fakeHash;
```
Comment (L-as) [inline src/app/zeko/default.nix]: This won't build. If you try building it will say the hash is incorrect, and then you can substitute the right hash in.

**Review by L-as — CHANGES_REQUESTED** [review]
Thanks

File: src/app/zeko/default.nix (hunk 1 above)
Comment (MartinOndejka) [reply src/app/zeko/default.nix]: When ran `nix build .?submodules=1` on our machine on this branch, it didn't complain.

Comment (L-as) [reply src/app/zeko/default.nix]:
```
[user@nixos:/work/mina]$ nix build .#mina-geth
warning: ignoring untrusted substituter 'https://storage.googleapis.com/mina-nix-cache', you are not a trusted user.
Run `man nix.conf` for more information on the `substituters` configuration option.
error: hash mismatch in fixed-output derivation '/nix/store/7zv9fz4a4skvy6wzi5x8an1i2ww6lj1k-source.drv':
         specified: sha256-AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA=
            got:    sha256-x7tY5Veh1fcoh/jMqBThJuQIkMuY2vUl9s/qRPM7soc=
error: 1 dependencies of derivation '/nix/store/ni1rba6gfhg4zzrj3mrpqb34my1zvzz3-go-ethereum-1.10.26.drv' failed to build
```

**Review by L-as — APPROVED** [review]

**Pull request merged**